src/gui/optionsdialog.ui

```diff
@@ -3606,17 +3606,31 @@ Specify an IPv4 or IPv6 address. You can specify &quot;0.0.0.0&quot; for any IPv
                  <property name="checked">
                   <bool>false</bool>
                  </property>
-                 <layout class="QHBoxLayout" name="horizontalLayout_5">
+                 <layout class="QVBoxLayout" name="verticalLayout_altWebUI">
                   <item>
-                   <widget class="QLabel" name="labelWebUIRootFolder">
+                   <layout class="QHBoxLayout" name="horizontalLayout_altWebUI_RootFolder">
+                    <item>
+                     <widget class="QLabel" name="labelWebUIRootFolder">
+                      <property name="text">
+                       <string>Files location:</string>
+                      </property>
+                     </widget>
+                    </item>
+                    <item>
+                     <widget class="FileSystemPathLineEdit" name="textWebUIRootFolder" native="true"/>
+                    </item>
+                   </layout>
+                  </item>
+                  <item>
+                   <widget class="QCheckBox" name="checkAltWebUIDerivative">
+                    <property name="toolTip">
+                     <string>Enables built-in translations and HTTPS security options. Indicates that the alternative Web UI is a derivative work of the original Web UI.</string>
+                    </property>
                     <property name="text">
-                     <string>Files location:</string>
+                     <string>Derived from original qBittorrent Web UI code base</string>
```

Comment:
I find such an option confusing.
Could you just add options that explicitly indicate what they are doing, such as "Enable server-side translation for alternative WebUI", etc.?